vaadinfrontend/src/main/java/life/qbic/datamanager/views/helloworld/HelloWorldView.java

```diff
@@ -16,7 +16,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 
 @PageTitle("Hello World")
-@Route(value = "", layout = MainLayout.class)
+@Route(value = "test", layout = MainLayout.class)
```

Comment:
test?

Reply:
That was a leftover from rerouting, I would suggest to remove this page, since we dont need it anymore. Any objections?